src/activity/events.jl

```diff
@@ -83,15 +83,15 @@ immutable EventListener
                 bt = catch_backtrace()
                 print(STDERR, "SERVER ERROR: ")
                 Base.showerror(STDERR, err, bt)
-                return HttpCommon.Response(500)
+                return HTTP.Response(500)
             end
         end
 
-        server.http.events["listen"] = port -> begin
-            println("Listening for GitHub events sent to $port;")
-            println("Whitelisted events: $(isa(events, Void) ? "All" : events)")
-            println("Whitelisted repos: $(isa(repos, Void) ? "All" : repos)")
-        end
+        #server.http.events["listen"] = port -> begin
```

Comment:
AFAIU, there is no way to do this with HTTP.jl right now.

Reply:
We could move the messages here into the `run(::EventListener, args...)` method below.

Reply:
True. Done.